modules/ROOT/pages/sap/sap-connector.adoc

```diff
@@ -36,74 +35,75 @@ and resources on external systems, such as Salesforce, SAP, ServiceNow, and Twit
 
 This document assumes you have:
 
-* A working knowledge of the SAP business context and in particular, the SAP R/3 Business Suite.
+* A working knowledge of the SAP business context and, in particular, the SAP R/3 Business Suite.
```

Comment:
I would remove periods at the ends of these list fragments, moving extra sentences below the list item....